### DISCUSSION TITLE
Welcome to KiCad-CustomDesignRules Discussions!

### DISCUSSION THREAD

**Discussion by Cimos** [Cimos/KiCad-CustomDesignRules#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ashway83** [comment]
Hi @Cimos,

I appreciate your efforts in picking up the development, as the author seems to have abandoned the original repository.
What I've noticed, though, is that your repository does not come up when searching for "kicad 8 rules," "kicad jlcpcb rules" and many other combinations. I believe you'll see more interest in your repo once search issue is fixed.

Cheers,
Andriy

**Reply by Cimos** [reply]
Hi @ashway83,

Thanks for the comment. I will look into this. 

If you have any suggestions that would be  helpful.

**Reply by Cimos** [reply]
@ashway83 

I had updated the tags for the repo a while ago and thought that it would fix up. But it looks like this might be the issue.

https://stackoverflow.com/questions/65244853/why-does-searching-in-a-forked-repo-on-github-show-no-result-while-searching-in/65244927#65244927

Edit: One option is to delete the fork and just make a new repo with it
